tests/reftests/deps-only.test

```diff
@@ -85,15 +85,15 @@ depends: "b"
 ### opam install a c --deps-only
 [ERROR] Package conflict!
   * Missing dependency:
-    - a
+    - deps-of-c -> b -> a
```

Comment:
For packages `deps-of-*`, it should be displayed differently. It is an internal package created by opam in the deps-only mechanism. Maybe `(deps of c)`?

Reply:
I agree, however the problem already existed before and i feel like it's a bit outside of the scope of this PR.
The virtual `deps-of-*` package is a hack introduced in `OpamClient` and getting rid of it seems tedious. I'd rather do that separately. If you're ok with that I'll open a new ticket to track it

Reply:
I was not proposing to remove the `deps-of-*` mechanism, but to display it differently as now it will be more shown. But yes, it can be an improvement of this RP that we can track with an issue.

Reply:
The thing is that the mechanism is linked to the way it's displayed. You cannot do one without the other as a package named `deps-of-something` is perfectly valid and cannot be distinguished from the `deps-of-*` virtual package hack